extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php

```diff
@@ -114,9 +111,6 @@ public function sendDigestToUser( $userID ) {
 
 		$params = [
 			'targetUser' => User::newFromId( $userID )->getName(),
-			'replyToAddress' => self::REPLY_ADDRESS,
-			'fromAddress' => self::FROM_ADDRESS,
-			'fromName' => self::FROM_NAME,
```

Comment:
These are now just using the defaults in the controller which have the same values.